packages/@react-aria/landmark/stories/Landmark.stories.tsx

```diff
@@ -337,14 +343,14 @@ function IframeExample() {
       });
 
       setTimeout(() => {
-        document.body.removeAttribute('data-react-aria-top-layer');
+        document?.body.removeAttribute('data-react-aria-top-layer');
       }, 100);
-    });
+    }) as EventListener);
 
     // When the iframe is re-focused, restore focus back inside where it was before.
     window.addEventListener('focus', () => {
       if (prevFocusedElement) {
-        prevFocusedElement.focus();
+        prevFocusedElement?.focus();
```

Comment:
we know it's not null here because of the if statement

Reply:
some of these may have been missed during a merge since I was adding strict while you were changing these files